routers/likes.py

```
@@ -18,36 +20,46 @@ def toggle_like(
-#
-# @router.delete("/posts/{post_id}/likes", response_model=LikesResponsePost)
-# def like_post(post_id: str, current_user: Annotated[dict, Depends(get_current_user)]):
-#     return delete_post_commet(post_id,current_user)
-#
-# @router.delete("/comments/{comment_id}/likes", response_model=LikesResponseComment)
-# def like_comment(comment_id: str, current_user: Annotated[dict, Depends(get_current_user)]):
-#     return delete_like_comment(comment_id,current_user)
\ No newline at end of file
+
+@router.get("/posts/{post_id}/likes", response_model=LikeResponse)
+def like_post(post_id: str,
```

Ruff (0.14.13)

Redefinition of unused `like_post` from line 20: `like_post` redefined here

(F811)